copper-swagger-api/src/main/java/com/pogeyan/swagger/api/utils/SwaggerHelpers.java

```diff
@@ -1344,4 +514,393 @@ private static String getB64Auth(String username, String password) {
+						List<BigInteger> value = convertInstanceOfObject(valueOfType, List.class);
+						serializeMap.put(var, value);
+					}
+
+				} else if (reqPropertyType.equals(PropertyType.BOOLEAN)) {
+					if (valueOfType instanceof Boolean) {
+						Boolean booleanValue = convertInstanceOfObject(valueOfType, Boolean.class);
+						serializeMap.put(var, booleanValue);
+					} else if (valueOfType instanceof List<?>) {
+						List<Boolean> booleanValue = convertInstanceOfObject(valueOfType, List.class);
+						serializeMap.put(var, booleanValue);
+					}
+
+				} else if (reqPropertyType.equals(PropertyType.DATETIME)) {
+
+					// SimpleDateFormat sdf = new SimpleDateFormat("E MMM dd
+					// HH:mm:ss Z yyyy", Locale.US);
+
+					if (valueOfType instanceof GregorianCalendar) {
+						Long value = convertInstanceOfObject(valueOfType, Long.class);
+						GregorianCalendar lastModifiedCalender = new GregorianCalendar();
+						lastModifiedCalender.setTimeInMillis(value);
+						serializeMap.put(var, lastModifiedCalender);
+					} else if (valueOfType instanceof List<?>) {
+						List<Long> value = convertInstanceOfObject(valueOfType, List.class);
+						List<GregorianCalendar> calenderList = new ArrayList<>();
+						value.forEach(v -> {
+							GregorianCalendar lastModifiedCalender = new GregorianCalendar();
+							lastModifiedCalender.setTimeInMillis(v);
+							calenderList.add(lastModifiedCalender);
+						});
+						serializeMap.put(var, calenderList);
+					}
+
+				} else if (reqPropertyType.equals(PropertyType.DECIMAL)) {
+					if (valueOfType instanceof Double) {
+						Double value = convertInstanceOfObject(valueOfType, Double.class);
+						serializeMap.put(var, value);
+					} else if (valueOfType instanceof List<?>) {
+						List<BigDecimal> value = convertInstanceOfObject(valueOfType, List.class);
+						serializeMap.put(var, value);
+					}
+				} else {
+					// string type
+					if (valueOfType instanceof String) {
+						String value = convertInstanceOfObject(valueOfType, String.class);
+						serializeMap.put(var, value);
+					} else if (valueOfType instanceof List<?>) {
+						List<String> value = convertInstanceOfObject(valueOfType, List.class);
+						serializeMap.put(var, value);
+					}
+				}
+			} else {
+				continue;
+			}
+		}
+		LOG.info("serializedMap:{}", serializeMap);
+		return serializeMap;
+	}
+
+	@SuppressWarnings("unused")
+	public static IRequest getImplClient(HttpServletRequest request) throws Exception {
+
+		String authorization = request.getHeader("Authorization");
+		String credentials[] = HttpUtils.getCredentials(authorization);
+		String pathFragments[] = HttpUtils.splitPath(request);
+		String method = request.getMethod();
+
+		Map<String, Object> inputMap = new HashMap<String, Object>();
+		Map<String, Object> requestBaggage = new HashMap<String, Object>();
+		Part filePart = null;
+		String jsonString = null;
+
+		SRequestMessage sRequestMessage = new SRequestMessage(credentials, pathFragments);
```

Comment:
Can we name SRequestMessage as something more generic? like RequestMessage is fine as well.